doc/interop-test-descriptions.md

```diff
@@ -517,6 +517,34 @@ Procedure:
 Asserts:
 * Call completed with status CANCELLED
 
+### timeout_on_sleeping_server
+
+This test verifies that an RPC request whose lifetime exceeds its configured timeout value will end with the DeadlineExceeded status.
+
+Server features:
+* [FullDuplexCall][]
+* [Compressable Payload][]
+* [Observe ResponseParameters.interval_us][]
+
+Procedure:
+ 1. Client calls FullDuplexCall with the following request and sets its timeout to 1ms.
```

Comment:
FullDuplexCall won't have any response unless one is requested. The interval_us is completely unnecessary it appears.